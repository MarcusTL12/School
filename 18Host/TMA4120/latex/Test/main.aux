\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode `"\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{OrganicChemistry}
\citation{OrganicChemistry}
\citation{OrganicChemistry}
\babel@aux{norsk}{}
\babel@aux{norsk}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Teori}{1}{section.1}}
\pgfsyspdfmark {pgfid3}{15615649}{30372631}
\pgfsyspdfmark {pgfid4}{18848289}{30372631}
\pgfsyspdfmark {pgfid5}{25823935}{30372791}
\pgfsyspdfmark {pgfid6}{29194474}{30372791}
\pgfsyspdfmark {pgfid2}{13104756}{30372631}
\pgfsyspdfmark {pgfid8}{13565962}{25642056}
\pgfsyspdfmark {pgfid9}{20189474}{25642056}
\pgfsyspdfmark {pgfid10}{25096388}{25442240}
\pgfsyspdfmark {pgfid7}{13324753}{25642056}
\pgfsyspdfmark {pgfid12}{11965363}{20911161}
\pgfsyspdfmark {pgfid13}{18740428}{20911161}
\pgfsyspdfmark {pgfid14}{23665299}{20911161}
\pgfsyspdfmark {pgfid15}{30302465}{20911161}
\pgfsyspdfmark {pgfid11}{11710500}{20911161}
\pgfsyspdfmark {pgfid17}{13851160}{16287160}
\pgfsyspdfmark {pgfid18}{16381206}{16380082}
\pgfsyspdfmark {pgfid19}{23825364}{16213174}
\pgfsyspdfmark {pgfid20}{26062318}{16380082}
\pgfsyspdfmark {pgfid16}{12358823}{16931596}
\pgfsyspdfmark {pgfid1}{13104756}{30372631}
\pgfsyspdfmark {pgfid21}{31491218}{30372631}
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Generell mekanisme for aldolkondensasjon\cite  {OrganicChemistry}\relax }}{1}{figure.1.1}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:genmek}{{1.1}{1}{Generell mekanisme for aldolkondensasjon\cite {OrganicChemistry}\relax }{figure.1.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Reaksjonsligninger}{1}{section.2}}
\pgfsyspdfmark {pgfid23}{13782908}{9001521}
\pgfsyspdfmark {pgfid24}{17455753}{9001521}
\pgfsyspdfmark {pgfid25}{30049265}{8777607}
\pgfsyspdfmark {pgfid26}{32844616}{8777607}
\pgfsyspdfmark {pgfid22}{10352168}{9001521}
\@writefile{toc}{\contentsline {section}{\numberline {3}Reaksjonsmekanismer}{2}{section.3}}
\pgfsyspdfmark {pgfid29}{13977535}{43914846}
\pgfsyspdfmark {pgfid32}{20144843}{43914846}
\pgfsyspdfmark {pgfid30}{20399706}{43914846}
\pgfsyspdfmark {pgfid33}{29430041}{44640514}
\pgfsyspdfmark {pgfid34}{35990111}{44640514}
\pgfsyspdfmark {pgfid28}{9411953}{43914846}
\pgfsyspdfmark {pgfid36}{13737609}{38123907}
\pgfsyspdfmark {pgfid37}{23335657}{38123907}
\pgfsyspdfmark {pgfid38}{31432101}{37730691}
\pgfsyspdfmark {pgfid35}{9172027}{38123907}
\pgfsyspdfmark {pgfid40}{12339047}{30143454}
\pgfsyspdfmark {pgfid41}{19886957}{30143454}
\pgfsyspdfmark {pgfid42}{28650479}{30143454}
\pgfsyspdfmark {pgfid45}{35952629}{30143454}
\pgfsyspdfmark {pgfid43}{36207492}{30143454}
\pgfsyspdfmark {pgfid39}{8908307}{30143454}
\pgfsyspdfmark {pgfid47}{14554876}{22556217}
\pgfsyspdfmark {pgfid48}{22249788}{22556217}
\pgfsyspdfmark {pgfid49}{24432531}{22492995}
\pgfsyspdfmark {pgfid50}{29859016}{22556217}
\pgfsyspdfmark {pgfid51}{37553928}{22556217}
\pgfsyspdfmark {pgfid46}{7848136}{23211417}
\pgfsyspdfmark {pgfid54}{12842693}{14968980}
\pgfsyspdfmark {pgfid53}{13977535}{14968980}
\pgfsyspdfmark {pgfid57}{20144843}{14968980}
\pgfsyspdfmark {pgfid55}{20399706}{14968980}
\pgfsyspdfmark {pgfid59}{28295199}{14968980}
\pgfsyspdfmark {pgfid58}{29430041}{14968980}
\pgfsyspdfmark {pgfid60}{35990111}{14968980}
\pgfsyspdfmark {pgfid52}{9411953}{14968980}
\pgfsyspdfmark {pgfid62}{14852311}{7289769}
\pgfsyspdfmark {pgfid63}{21524957}{7289769}
\pgfsyspdfmark {pgfid64}{31682899}{7289769}
\pgfsyspdfmark {pgfid67}{38109785}{7289769}
\pgfsyspdfmark {pgfid65}{38364648}{7289769}
\pgfsyspdfmark {pgfid61}{6751151}{7891946}
\pgfsyspdfmark {pgfid69}{15430140}{-587462}
\pgfsyspdfmark {pgfid70}{22249788}{-587462}
\pgfsyspdfmark {pgfid71}{24432531}{-703707}
\pgfsyspdfmark {pgfid72}{31253436}{-587462}
\pgfsyspdfmark {pgfid73}{38073084}{-587462}
\pgfsyspdfmark {pgfid68}{7328980}{14715}
\pgfsyspdfmark {pgfid27}{9411953}{43914846}
\pgfsyspdfmark {pgfid74}{39553401}{43914846}
\@writefile{lof}{\contentsline {figure}{\numberline {3.1}{\ignorespaces Mekanisme for reaksjonen\relax }}{2}{figure.3.1}}
\newlabel{fig:mek}{{3.1}{2}{Mekanisme for reaksjonen\relax }{figure.3.1}{}}
\citation{SI}
\citation{SI}
\@writefile{toc}{\contentsline {section}{\numberline {4}Fysikalske Data}{3}{section.4}}
\@writefile{lot}{\contentsline {table}{\numberline {4.1}{\ignorespaces Relevante molare masser\cite  {SI}\relax }}{3}{table.4.1}}
\newlabel{tab:fysdat}{{4.1}{3}{Relevante molare masser\cite {SI}\relax }{table.4.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Eksperimentelt}{3}{section.5}}
\@writefile{toc}{\contentsline {section}{\numberline {6}Resultater}{3}{section.6}}
\@writefile{lot}{\contentsline {table}{\numberline {6.1}{\ignorespaces Resultater for syntese\relax }}{3}{table.6.1}}
\newlabel{tab:res}{{6.1}{3}{Resultater for syntese\relax }{table.6.1}{}}
\bibstyle{ieeetr}
\bibdata{bibliografi}
\bibcite{OrganicChemistry}{{1}{}{{}}{{}}}
\bibcite{SI}{{2}{}{{}}{{}}}
\@writefile{toc}{\contentsline {section}{\numberline {7}Diskusjon}{4}{section.7}}
\providecommand\NAT@force@numbers{}\NAT@force@numbers
