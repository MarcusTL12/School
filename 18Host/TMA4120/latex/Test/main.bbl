\begin{thebibliography}{1}

\bibitem{OrganicChemistry}
{Solomons Graham, T.W., Fryhle B. Craig, Snyder A. Scott}, {\em SolomonsÂ´
  Organic Chemistry}.
\newblock Wiley, 12th~ed., 2016.
\newblock s. 856-859\EatDot.

\bibitem{SI}
A.~G. Blackman, {\em Aylward and Findlay's SI chemical data}.
\newblock Wiley, 7th~ed., 2014\EatDot.

\end{thebibliography}
