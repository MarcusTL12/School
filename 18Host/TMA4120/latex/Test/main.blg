This is BibTeX, Version 0.99d (TeX Live 2018/W32TeX)
Capacity: max_strings=100000, hash_size=100000, hash_prime=85009
The top-level auxiliary file: main.aux
The style file: ieeetr.bst
Database file #1: bibliografi.bib
You've used 2 entries,
            1876 wiz_defined-function locations,
            491 strings with 3690 characters,
and the built_in function-call counts, 337 in all, are:
= -- 33
> -- 8
< -- 0
+ -- 4
- -- 2
* -- 16
:= -- 57
add.period$ -- 5
call.type$ -- 2
change.case$ -- 2
chr.to.int$ -- 0
cite$ -- 2
duplicate$ -- 18
empty$ -- 41
format.name$ -- 2
if$ -- 82
int.to.chr$ -- 0
int.to.str$ -- 2
missing$ -- 4
newline$ -- 12
num.names$ -- 2
pop$ -- 9
preamble$ -- 1
purify$ -- 0
quote$ -- 0
skip$ -- 2
stack$ -- 0
substring$ -- 0
swap$ -- 2
text.length$ -- 0
text.prefix$ -- 0
top$ -- 0
type$ -- 0
warning$ -- 0
while$ -- 2
width$ -- 3
write$ -- 24
