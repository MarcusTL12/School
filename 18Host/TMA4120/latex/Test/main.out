\BOOKMARK [1][-]{section.1}{Teori}{}% 1
\BOOKMARK [1][-]{section.2}{Reaksjonsligninger}{}% 2
\BOOKMARK [1][-]{section.3}{Reaksjonsmekanismer}{}% 3
\BOOKMARK [1][-]{section.4}{Fysikalske Data}{}% 4
\BOOKMARK [1][-]{section.5}{Eksperimentelt}{}% 5
\BOOKMARK [1][-]{section.6}{Resultater}{}% 6
\BOOKMARK [1][-]{section.7}{Diskusjon}{}% 7
